<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageFrame</class>
 <widget class="QWidget" name="ImageFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image tools</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QToolButton" name="insertImageButton">
     <property name="toolTip">
      <string>Insert inline image</string>
     </property>
     <property name="text">
      <string>Add image</string>
     </property>
     <property name="icon">
      <iconset resource="pdf.qrc">
       <normaloff>:/images/addImage.png</normaloff>:/images/addImage.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="selectImageButton">
     <property name="toolTip">
      <string>select inline image</string>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
     <property name="icon">
      <iconset resource="pdf.qrc">
       <normaloff>:/images/hand.png</normaloff>:/images/hand.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteImageButton">
     <property name="toolTip">
      <string>delete inline image</string>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset resource="pdf.qrc">
       <normaloff>:/images/DeleteText.png</normaloff>:/images/DeleteText.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="changeImageButton">
     <property name="toolTip">
      <string>change inline image</string>
     </property>
     <property name="text">
      <string>Change</string>
     </property>
     <property name="icon">
      <iconset resource="pdf.qrc">
       <normaloff>:/images/change.png</normaloff>:/images/change.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="imagePartButton">
     <property name="toolTip">
      <string>extract part of pdf as image</string>
     </property>
     <property name="text">
      <string>SnapShot</string>
     </property>
     <property name="icon">
      <iconset resource="pdf.qrc">
       <normaloff>:/images/snapshot.png</normaloff>:/images/snapshot.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="extractButton">
     <property name="toolTip">
      <string>extract inline image</string>
     </property>
     <property name="text">
      <string>ShapShot
(imageOnly)</string>
     </property>
     <property name="icon">
      <iconset resource="pdf.qrc">
       <normaloff>:/images/snapshot.png</normaloff>:/images/snapshot.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>16</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="pdf.qrc"/>
 </resources>
 <connections/>
</ui>
